bitnami/argo-workflows/Chart.yaml

```diff
@@ -35,4 +35,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/argo-workflow-controller
   - https://github.com/bitnami/containers/tree/main/bitnami/argo-workflow-exec
   - https://argoproj.github.io/workflows/
-version: 5.1.0
+version: 5.1.1
```

Comment:
```suggestion
version: 5.1.2
```

Reply:
@CeliaGMqrz Done. Thanks for the review!